integration/yarn/src/main/java/alluxio/yarn/Client.java

```diff
@@ -342,52 +326,4 @@ private void setupApplicationSubmissionContext() {
     // Set the priority for the application master
     mAppContext.setPriority(Priority.newInstance(mAmPriority));
   }
-
```

Comment:
the only progress we were monitoring before was whether or not the AM crashed, which is very uncommon because it isn't doing much.